### ISSUE TITLE
Failed to install module php:remi-8.0

### ISSUE THREAD

**Issue by Richard87** [remicollet/remirepo#172]
Hi!

I'm trying to install php80 on a fresh Fedora34(rc1) installation right now, but I have some dependency errors on this repository:

```
root@w ➜  dnf config-manager --set-enabled remi
root@w ➜  dnf config-manager --set-enabled remi-php80
root@w ➜  dnf update
Remi's RPM repository - PHP 8.0 - Fedora 34 - x86_64                                                                                                                          13 kB/s | 3.0 kB     00:00    
Remi's RPM repository - Fedora 34 - x86_64                                                                                                                                    15 kB/s | 3.0 kB     00:00    
Dependencies resolved.
Nothing to do.
Ferdig!

root@w ➜  dnf module install php:remi-8.0
Last metadata expiration check: 0:01:10 ago on ti. 27. april 2021 kl. 11.48 +0200.
All matches for argument 'php:remi-8.0' in module 'php:remi-8.0' are not active
Feil: Problems in request:
broken groups or modules: php:remi-8.0
Modular dependency problems:

 Problem: conflicting requests
  - nothing provides module(platform:f33) needed by module php:remi-8.0:20210426054307:00000000.x86_64

root@w ➜  richard dnf install php-cli 
Last metadata expiration check: 0:00:48 ago on ti. 27. april 2021 kl. 11.48 +0200.
Dependencies resolved.
=============================================================================================================================================================================================================
 Package                                           Architecture                                  Version                                                 Repository                                     Size
=============================================================================================================================================================================================================
Installing:
 php-cli                                           x86_64                                        7.4.16-1.fc34                                           fedora                                        4.6 M
Installing dependencies:
 php-common                                        x86_64                                        7.4.16-1.fc34                                           fedora                                        793 k

Transaksjonsammendrag
=============================================================================================================================================================================================================
Install  2 Packages

Totale størrelse på pakker som hentes: 5.4 M
Installed size: 37 M
Er dette ok [j/N]: N
Operation aborted.

```

**Comment by Richard87** [comment]
I manually installed if from the mirro, which works nice :) 

```
dnf install http://rpms.remirepo.net/fedora/34/modular/x86_64/php-8.0.3-1.fc34.remi.x86_64.rpm http://rpms.remirepo.net/fedora/34/modular/x86_64/php-8.0.3-1.fc34.remi.x86_64.rpm http://rpms.remirepo.net/fedora/34/modular/x86_64/php-cli-8.0.3-1.fc34.remi.x86_64.rpm http://rpms.remirepo.net/fedora/34/modular/x86_64/php-common-8.0.3-1.fc34.remi.x86_64.rpm ....
```

**Comment by o-alquimista** [comment]
> I manually installed if from the mirro, which works nice :)
> http://rpms.remirepo.net/fedora/34/modular/x86_64
> 
> ```
> dnf install \
>     http://rpms.remirepo.net/fedora/34/modular/x86_64/php-8.0.3-1.fc34.remi.x86_64.rpm \
>     http://rpms.remirepo.net/fedora/34/modular/x86_64/php-cli-8.0.3-1.fc34.remi.x86_64.rpm \
>     http://rpms.remirepo.net/fedora/34/modular/x86_64/php-common-8.0.3-1.fc34.remi.x86_64.rpm \
>     ...
> ```

Will that use automatic updates, though?

I have the same issue. Would be nice to open this issue again.

**Comment by Richard87** [comment]
Nope, probably not, will have to come back to it when 8.0.4 is released :)

**Comment by remicollet** [comment]
> Nope, probably not, will have to come back to it when 8.0.4 is released :)

8.0.4 will never exists, see #173 which is now fixed (and allow to install 8.0.5, to be "officially" announced tomorrow)

**Comment by Richard87** [comment]
Wow, thanks for the quick fix! :+1:
